ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -311,8 +311,11 @@ private void prepareReturnValues(List<String> values) throws SemanticException {
   }
 
   private void deleteAllPreviousDumpMeta(Path currentDumpPath) {
+    Path dumpRoot = getDumpRoot(currentDumpPath);
```

Comment:
Do it inside the try catch block. replication shouldn't fail if getDumpRoot throws exception